packages/utils/src/cmds/release.js

```diff
@@ -116,9 +117,7 @@ const handler = async ({ publish }) => {
             const { lastRelease, commits, nextRelease, releases } = result
 
             reporter.info(
-                `Published ${nextRelease.type} release version ${
-                    nextRelease.version
-                } containing ${commits.length} commits.`
+                `Published ${nextRelease.type} release version ${nextRelease.version} containing ${commits.length} commits.`
```

Comment:
Prettier changed behaviour on how it splits template literals.

Reply:
👍 I like this way better